PlayFabClientSDK/source/PlayFabTransport.cs

```diff
@@ -0,0 +1,44 @@
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using PlayFab.Internal;
+
+namespace PlayFab
+{
+    /// <summary>
+    /// Sends request data.
+    /// </summary>
+    public class PlayFabTransport: ITransportPlugin
+    {
+        private static readonly IPlayFabHttp _http;
+
+        static PlayFabTransport()
+        {
+            var httpInterfaceType = typeof(IPlayFabHttp);
+            var types = typeof(PlayFabHttp).GetAssembly().GetTypes();
+            foreach (var eachType in types)
+            {
+                if (httpInterfaceType.IsAssignableFrom(eachType) && !eachType.IsAbstract)
+                {
+                    _http = (IPlayFabHttp)Activator.CreateInstance(eachType.AsType());
```

Comment:
Same thing with JsonWrapper.
IPlayFabHttp should just be replaced with this wrapper.
Double wrapping these doesn't make sense.